tests/ert/unit_tests/gui/simulation/test_run_dialog.py

```diff
@@ -115,7 +115,14 @@ def mock_get_runtime():
 
 
 @pytest.fixture
-def run_dialog(qtbot: QtBot, use_tmpdir, storage):
+def mock_set_env_key():
```

Comment:
Does this mean that we dont need to set any env variable to run EnsembleExperiment?